src/main/java/Command.java

```diff
@@ -0,0 +1,8 @@
+
+public abstract class Command {
```

Comment:
I like this abstraction and the use of inheritance! Makes the command handling very clean.